src/hdcontroller.ts

```diff
@@ -78,7 +80,7 @@ export class HDProxydClient {
     private path: string;
     public bootstrap: string[][];
     private httpAgent: http.Agent;
-    private logging: RequestLogger | any;
+    private logging: werelogs.Logger | any;
```

Comment:
We should also change the `logApi` type above as `typeof werelogs`, as it's `any` currently, but we only support a werelogs object (or werelogs-compatible)